core/language/en-GB/Misc.multids

```diff
@@ -39,6 +39,9 @@ Error/XMLHttpRequest: XMLHttpRequest error code
 InternalJavaScriptError/Title: Internal JavaScript Error
 InternalJavaScriptError/Hint: Well, this is embarrassing. It is recommended that you restart TiddlyWiki by refreshing your browser
 InvalidFieldName: Illegal characters in field name "<$text text=<<fieldName>>/>". Fields can only contain lowercase letters, digits and the characters underscore (`_`), hyphen (`-`) and period (`.`)
+LayoutSwitcher/Description: open the layout switcher
+LayoutSwitcher/Hint: Layout
+LayoutSwitcher/SubTitle: Layout Switcher
```

Comment:
I'm unsure about describing it as a "layout switcher" since it lets you switch more than just the layout, i.e. themes, languages etc. @Jermolene thoughts?

Reply:
Ideally the modal should be captioned according to the type of thing that is being switched: "Theme Switcher", "Layout Switcher" etc.

Reply:
@Jermolene, I don't know how to do that, do you have an idea?

Reply:
@BurningTreeC see my comment at https://github.com/Jermolene/TiddlyWiki5/pull/5003/files#r519956303

We would also want to choose which tab to display accordingly which is where it gets trickier I think.